src/walltime.cc

```diff
@@ -147,7 +167,8 @@ WallTimeImp::WallTimeImp()
       cycles_per_second_(0), seconds_per_cycle_(0.0),
       last_adjust_time_(0), drift_adjust_(0),
       max_interval_cycles_(0) {
-  cycles_per_second_ = static_cast<int64_t>(CyclesPerSecond());
+    const double kMaxErrorInterval = 100e-6;
```

Comment:
>
> is the indent off here or is github playing tricks on me?
>
Probably me.

What should I do? Amend my commits according to your remarks, force push to
my repo and submit a new PR? Or create one (or several) corrective commits?

J-L